src/pint/modelutils.py

```diff
@@ -2,9 +2,17 @@
 
 
 def convert_to_equatorial(model):
-    """Converts pulsar's PulsarEcliptic coordinates to ICRS,
-    adds the AstrometryEquatorial model component with the
-    correct values, and removes AstrometryEcliptic component.
+    """Converts Astrometry model component, Ecliptic to Equatorial
+
+      Parameters
+      ----------
+      model
```

Comment:
After the parameter name, giving a required type would be helpful, like:
```
model: `pint.models.TimingModel` object
```

Reply:
I can certainly do that, although I made this decision by going and finding/copying other examples like this in existing code. So it might be good to impose a standard if anyone else, like me, looks for examples this way.

Reply:
This has been addressed.